website/src/pages/plugins/presets/preset-client.mdx

```diff
@@ -525,3 +526,82 @@ const nextConfig = {
 ```
 
 Note that you will need to provide the `artifactDirectory` path that should be the same as the one configured in your `codegen.ts`
+
+### DocumentMode
```

Comment:
Is it the right place for this in the docs?

Reply:
yeah should be good here. Maybe we should have some FAQ section or something that will tell why one would prefer using this option 🤔

Reply:
@beerose Yeah, I agree, having a section for when to use this over typed document node makes sense!

Reply:
I added a short section in the same file